auth/README.md

```diff
@@ -204,8 +204,8 @@ startActivityForResult(
 
 #####Response codes
 The authentication flow only provides three response codes:
```

Comment:
This sentence is no longer true ("only three response codes..."). Maybe we should just point to the ResponseCodes file here and rely on JavaDoc comments so we don't have to keep updating this.